### DISCUSSION TITLE
User with Manager role gets "401 Unauthorized" when accessing Organization Manage tab

### DISCUSSION THREAD

**Discussion by Dimonina** [dani-garcia/vaultwarden#2234] [Q&A]
### Subject of the issue
The manager role does not to seem work well. As a manager, I can't manage Organization's collections.

### Deployment environment
* vaultwarden version: 1.23.1
* Install method: docker
* Clients used: web
* Reverse proxy and version: no proxy

### Steps to reproduce
1. Create first admin user "admin@test.com"
2. Create an organization under this user, e.g. "MyOrg"
3. Create a couple of collections "Col1", "Col2", "Col2/Col2-1"
4. Create a new user "manager@test.com"
5. Assign a manager role to manager@test.com under admin user and give rights to "Col2", "Col2/Col2-1" collection.
6. Under manager@test.com using web interface go to organization and click "Manage" tab.
### Expected behaviour
User can see a page with an ability to manage collections

### Actual behaviour
User is logged out

**Comment by BlackDex** [comment]
Duplicate of #2151 
And already fixed in #2169

**Comment by Dimonina** [comment]
> Duplicate of #2151 And already fixed in #2169

Mmm. If it is fixed, then why I can reproduce it in the latest container version?

**Reply by BlackDex** [reply]
Because `latest` is linked to tagged released images.
The `main` branch is available via the `testing` tagged images.
